test-cases/tests/alerts/c13b-verify-rhoam-reconciliation-alerts-are-not-firing.md

```diff
@@ -0,0 +1,30 @@
+---
+products:
+  - name: rhoam
+    environments:
+      - osd-post-upgrade
+      - osd-fresh-install
+    targets:
```

Comment:
@psturc do you think this one should be per-release?

Reply:
This test case now says that it should `Verify RHOAMnstallationControllerIsInReconcilingErrorState alerts are present and are not firing`

I think that the first part (check that alert is present) should be covered by [the automated test](https://github.com/integr8ly/integreatly-operator/pull/1519/files#diff-91e53f30685bc5065b0c3501fc4c08dc7a2ed0778429cdea3c77d04d9bbd330bR248)
The second part (alert is not firing) is already covered by [this test case](https://github.com/integr8ly/integreatly-operator/blob/fcc80cc5080db8a8a9e3d022f755e7ae66ab5ace/test-cases/tests/alerts/c05b-verify-alerts-are-exposed-and-no-critical-alerts-have-fired.md)

So maybe c13b test case is not needed anymore in the end?

Reply:
Even better! The alert is also changed for RHMI so perhaps we could remove both test cases?

Reply:
I believe so

Reply:
this is a [warning level alert](https://github.com/integr8ly/integreatly-operator/pull/1519/files#diff-7d10ed51e49fd1fef8a5c0a2f1074fd700809f789237a1dd39cfe136e5655d4fL45) so don't think it will be covered by [this test case](https://github.com/integr8ly/integreatly-operator/blob/fcc80cc5080db8a8a9e3d022f755e7ae66ab5ace/test-cases/tests/alerts/c05b-verify-alerts-are-exposed-and-no-critical-alerts-have-fired.md)

Reply:
actually having read the test case , only the title mentions critical alerts, the test case checks for any alert firing , happy to remove these